quick-start-guides/how-to-become-a-referrer.md

```diff
@@ -0,0 +1,38 @@
+# How to become a Referrer?
+
+## Step 1: Getting Started
+
+If this is your first time on https://app.buffer.finance/, before you start placing up/down trades, follow the "Getting started" that will help you with connecting to the dApp, and preparing your wallet with sufficient ETH for gas among other pre-requisites.
+
+## Step 2: Create a referral code
+
+To participate in the referral program, first, you need to create a referral code. To do that, navigate to https://app.buffer.finance/#/referral.
+
+Click on  **'Create your Referral'** and enter any combination of letters, numbers, and underscores to create a referral code. Click on **'Create'**.
+
+<img width="871" alt="Screenshot 2023-03-15 at 5 41 30 PM" src="https://user-images.githubusercontent.com/126849045/225305621-1fe704d6-093b-4242-a3a5-f527ffc41ab2.png">
+
+_Note that referral codes are case-sensitive and that your code must be created on both Arbitrum as well as Polygon to earn trader rewards on both networks._
+
+Once the referral code is successfully created,  click on the copy icon next to the code to get your referral link, it should look something like this: https://app.buffer.finance/#/?ref='your_code_here'.
+
+Once copied, you can share your referral link on any social media or communication platform. When a user clicks on your link, your referral code will be connected to their account. When a user (referee) under your referral network, makes a trade they will receive a payout boost depending on their tier and you (referer) will receive trader rewards on the referee's trade size. 
+
+When the user makes a trade they would receive a discount and you would earn rebates from their trading fees. You will continue to earn rebates even if the user uses another device later on as the referral code is stored on the contract the first time the user makes a trade.
```

Comment:
The referral code is stored for the refree only if the user explicitly sets it but if we directly comes through a referral link then the referral code is applied but not saved in the contract. So if the user uses another device then the referral code wont apply automatically
@RizzJ

Reply:
So if the user on new/different devices, enters the referral code, using the "use referral code" function, will then the referer start earning rebates on trades made on that account?

@bufferfinance